activetable.xml

```diff
@@ -145,8 +160,24 @@ table = [
     ['purple', 3.4, 2.1, Numeric(answer=-1.3)],
 ]
 
+# Set the text that gets displayed when clicking the "+help" button.  Use "None" to disable the
+# help feature.
+#help_text = None
+help_text = "Fill in the yellow fields with the right values."
+
+# Set the width of the columns in pixels.  The total width of the table shouldn't be more than 800.
+# You can use "column_width = None" for equal-width columns with a total width of 800 pixels.
+column_widths = None
+#column_widths = [150, 150, 150, 150]
+
+# The height of a table row in pixels.  Rows may grow higher than the specified value if the text
+# in some cells in the row is long enough to get wrapped in more than more lines.  If this happens,
```

Comment:
@smarnach probably typo: "more than more lines". At least my lexical parser raised an exception trying to parse this :)